manifests/m/Microsoft/DSC/Preview/3.1.0-preview.1/Microsoft.DSC.locale.en-US.yaml

```diff
@@ -0,0 +1,41 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Microsoft.DSC.Preview
+PackageVersion: 3.1.0-preview.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://github.com/PowerShell
+PublisherSupportUrl: https://github.com/PowerShell/DSC/issues
+PackageName: DSC v3 Preview
+PackageUrl: https://github.com/PowerShell/DSC
+License: MIT
+LicenseUrl: https://github.com/PowerShell/DSC/blob/main/LICENSE
+Copyright: Copyright (c) Microsoft Corporation
+CopyrightUrl: https://github.com/PowerShell/DSC/blob/main/LICENSE
+ShortDescription: DSCv3 is an open source command line application that abstracts the management of software components declaratively and idempotently.
+Description: |-
+  DSCv3 is the latest iteration of Microsoft's Desired State Configuration platform.
+  DSCv3 is an open source command line application that abstracts the management of software components declaratively and idempotently.
+  DSCv3 runs on Linux, macOS, and Windows without any external dependencies.
+
+  With DSCv3, you can:
+
+  - Author resources to manage your systems in any language
+  - Invoke individual resources
+  - Create configuration documents that define the desired state of a system
+Moniker: dscv3-preview
+Tags:
+- dsc
+- dscv3-preview
```

Comment:
```suggestion
Tags:
- dsc
- dscv3
- dscv3-preview
```

Looks good, just one minor comment. I would still want this package to appear based on the tags field when someone does `winget search dscv3`

![image](https://github.com/user-attachments/assets/4dc8065b-6092-48a6-9040-3e588112dea2)